pkg/controller/user-cluster-controller-manager/resources/resources/core-dns/deployment.go

```diff
@@ -104,6 +108,22 @@ func DeploymentCreator(kubernetesVersion *semverlib.Version, registryWithOverwri
 
 			dep.Spec.Template.Spec.ServiceAccountName = resources.CoreDNSServiceAccountName
 
+			dep.Spec.Template.Spec.Affinity = &corev1.Affinity{
+				PodAntiAffinity: &corev1.PodAntiAffinity{
+					PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+						{
+							Weight: 10,
```

Comment:
I think this weight is not necessary here. I know it was probably copied from elsewhere, but still ;)